components/filter/test/filter.axe.js

```diff
@@ -32,6 +32,8 @@ describe('d2l-filter', () => {
 		it(`${test.name} opened`, async() => {
 			const elem = await fixture(test.fixture);
 			const dropdown = elem.shadowRoot.querySelector('d2l-dropdown-button-subtle');
+			const dropdownContent = elem.shadowRoot.querySelector('d2l-dropdown-content') || elem.shadowRoot.querySelector('d2l-dropdown-menu');
```

Comment:
Single and multiple dimension filters use content and menu, respectively. Waiting for the update of the content here, like other tests.